packages/core/src/components/Dropdown/index.tsx

```diff
@@ -105,7 +89,7 @@ class Dropdown extends React.PureComponent<Props & WithStylesProps> {
       onClickOutside,
       ...props
     } = this.props;
-    const style: React.CSSProperties = {
```

Comment:
why wasn't this using withStyles before?

Reply:
It was, just the type was wrong.

Reply:
aah i expanded the rest of the diff and i see it now.

Reply:
😎